acis_thermal_check/state_builder.py

```diff
@@ -256,8 +465,26 @@ def get_validation_states(self, datestart, datestop):
         datestop = DateTime(datestop).date
         self.logger.info('Getting commanded states between %s - %s' %
                          (datestart, datestop))
-        return cmd_states.fetch_states(datestart, datestop)
```

Comment:
It looks like now that `get_validation_states` is the same for both `StateBuilder` subclasses. We should probably move this up to the superclass then (which I now realize requires a small refactor of `__init__` also).